docs/content/Examples-Tutorials-Recipes/Active-users.md

```diff
@@ -8,88 +8,119 @@ redirect_from:
+## Data schema
 
-To calculate daily, weekly, or monthly active users we’re going to use the
-`rollingWindow` measure parameter. `rollingWindow` accepts 3 parameters:
-trailing, leading, and offset. You can read about what each of them does
-[here](/schema/reference/measures#parameters-rolling-window).
-
-For our purpose, we need only offset and trailing. We will set offset to
-<b>start</b> and the trailing parameter to the number of days – 1, 7, or 30.
+Daily, weekly, and monthly active users are commonly referred to as DAU, WAU,
+MAU. To get these metrics, we need to use a rolling time frame to calculate a
+daily count of how many users interacted with the product or website in the
+prior day, 7 days, or 30 days. Also, we can build other metrics on top of these
+basic metrics. For example, the WAU to MAU ratio. We can add it, using already
+defined `weeklyActiveUsers` and `monthlyActiveUsers`.
 
-In the example below, we’ll create a cube called `ActiveUsers` with data from
-our events table.
-
-<div class="block help-block">Please note, we are using interval literal in the trailing parameter.
-The example below should work in Redshift and BigQuery. The exact interval literal could be different, depending on your database.
-</div>
+To calculate daily, weekly, or monthly active users we’re going to use the
+[`rollingWindow`](https://cube.dev/docs/schema/reference/measures#parameters-rolling-window)
+measure parameter.
 
 ```javascript
-cube(`ActiveUsers`, {
-  sql: `select id, user_id, timestamp from events`,
+cube(`Users`, {
+  sql: `SELECT * FROM public.users`,
+
   measures: {
     monthlyActiveUsers: {
-      sql: `user_id`,
+      sql: `id`,
       type: `countDistinct`,
       rollingWindow: {
         trailing: `30 day`,
         offset: `start`,
       },
     },
+
     weeklyActiveUsers: {
-      sql: `user_id`,
+      sql: `id`,
       type: `countDistinct`,
       rollingWindow: {
-        trailing: `1 week`,
+        trailing: `7 day`,
         offset: `start`,
       },
     },
+
     dailyActiveUsers: {
-      sql: `user_id`,
+      sql: `id`,
       type: `countDistinct`,
       rollingWindow: {
         trailing: `1 day`,
         offset: `start`,
       },
     },
+
+    wauToMau: {
+      title: `WAU to MAU`,
+      sql: `100.000 * ${weeklyActiveUsers} / NULLIF(${monthlyActiveUsers}, 0)`,
+      type: `number`,
+      format: `percent`,
+    },
   },
+
   dimensions: {
-    timestamp: {
-      sql: `timestamp`,
+    createdAt: {
+      sql: `created_at`,
       type: `time`,
     },
   },
 });
 ```
 
-Going further, we can build other metrics on top of these basic metrics. For
-example, <b>the DAU to MAU ratio</b> is one of the most popular metrics used to
-<b>measure the stickiness of the product</b>. We can easily add it, using
-already defined `dailyActiveUsers` and `monthlyActiveUsers`.
+## Query
+
+We should set a `timeDimensions` with the `dateRange` in the query that will
```

Comment:
Should this be "... in the query that will be sent to the Cube API"?

Reply:
Yeah, I guess we can delete that :)